tests.py

```diff
@@ -18,27 +17,7 @@
 from xml.etree import ElementTree as ET
 
 
-if sys.version_info >= (3, 8):
-    from unittest import mock
-else:
-    # unittest.mock in 3.7 is too old to support
-    # all the features used in the test suite
-    import mock
-
-from check_manifest import rmtree
```

Comment:
This import is necessary and removing it breaks the test suite (which you can only see on Appveyor because GitHub Actions cancels all the useful test runs early after seeing the isort failure).